### PR TITLE
33, 35, 36 - choice of images; flaps and foldouts; composition from multiple images

### PR DESCRIPTION
**Pull request by tomcrane**
Not sure if this is quite right yet...

Dawn's update
Previews:
https://preview.iiif.io/cookbook/3333-choice/recipe/0033-choice/
https://preview.iiif.io/cookbook/3333-choice/recipe/0035-foldouts/
https://preview.iiif.io/cookbook/3333-choice/recipe/0036-composition-from-multiple-images/

### CODE REVIEW

**Comment by glenrobson** [comment]
(Automated comment): Branch aviliable for testing at https://preview.iiif.io/cookbook/3333-choice

**Comment by azaroth42** [comment]
Needs a new image, but otherwise is good. Can replace the image later?

File: recipe/0033-choice/manifest.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: null
+---
+{
+    "@context": "http://www.w3.org/ns/anno.jsonld",
```
Comment (azaroth42) [inline recipe/0033-choice/manifest.json]: This is the wrong context

**Review by azaroth42 — DISMISSED** [review]
Sorry, context is the wrong one.

**Comment by tomcrane** [comment]
Context updated... (when going from 2 contexts to one, pay attention to which one I'm throwing away)

**Comment by tomcrane** [comment]
I couldn't find a "licensed for non commercial reuse" image of a multispectral set or _stack_, like this one. If necessary someone could make a new image but there must be one out there. I think it really helps understand the point of the recipe and the relation of images to Canvas.

**Comment by mikeapp** [comment]
Preview direct link to recipe: https://preview.iiif.io/cookbook/3333-choice/recipe/0033-choice/

**Comment by tomcrane** [comment]
This latest commit aims for a coherent story across the multi-image use cases.
While #33 and #34 go together (34 is just 33 + image services), #35 (Foldouts) and #36 (Multiple images and composition) are separate, and I can break them out into separate issues if necessary. This PR was started for the Ann Arbor walkthrough where I was looking at several recipes.

They all reference and discuss each other, and I think they should be considered by the TRC together as well, otherwise we may end up with inconsistent approaches to modelling scenarios that involve more than one content resource associated with the same _part_ of an object.

* In #33 and #34, the content resources are there to generate a UI for the user to make a choice.
* In #35 the content resources might be photographs of the same physical part (e.g., a map in a book) but targeted at different views (and therefore different canvases).
* #36 is about scene composition - _Fire_ would also be a good example to add here, later.

**Comment by tomcrane** [comment]
(Thank you for the validator...!)

Links for convenience:

* https://preview.iiif.io/cookbook/3333-choice/recipe/0033-choice/
* https://preview.iiif.io/cookbook/3333-choice/recipe/0034-choice-with-image-services/
* https://preview.iiif.io/cookbook/3333-choice/recipe/0035-foldouts/
* https://preview.iiif.io/cookbook/3333-choice/recipe/0036-composition-from-multiple-images/

**Comment by glenrobson** [comment]
Hi Tom,

Not sure if your still using the dee images but they are now available as straight images as well as IIIF images see for example the following:

https://fixtures.iiif.io/info.html?file=/images/Wellcome/dee-natural.jpg

Which has the following straight image link:

https://fixtures.iiif.io/images/Wellcome/dee-natural.jpg

File: recipe/0036-composition-from-multiple-images/manifest.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                          "height":2414,
+                          "service": [
+                            {
+                              "id": "https://gallica.bnf.fr/iiif/ark:/12148/btv1b10511139b/f1",
+                              "type": "ImageService1",
+                              "profile": "level2"
+                            }
+                          ]
+                        },
+                        "target": "https://bvmm.irht.cnrs.fr/iiif/2309/canvas/canvas-981394#xywh=3949,994,1091,1232"
```
[the remark this answers is not in the document]
Comment (kirschbombe) [reply recipe/0036-composition-from-multiple-images/manifest.json:57]: Hi, @stephenwf - thanks for the suggested change. These manifests are quite outdated and still a work in progress. The current images will all be replaced with ones on the IIIF fixtures service when I get back to editing this selection of recipes later in December :)

[the remark this answers is not in the document]
Comment (stephenwf) [reply recipe/0036-composition-from-multiple-images/manifest.json:57]: Ah, no worries!

File: recipe/0036-composition-from-multiple-images/manifest.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+    "@context": "http://iiif.io/api/presentation/3/context.json",
+    "id": "{{ id.url }}",
+    "type": "Manifest",
+    "label": { "en": [ "Folio from Grandes Chroniques de France, ca. 1460" ] },
+    "items": [
+        {
+            "id": "https://bvmm.irht.cnrs.fr/iiif/4490/canvas/p1",
```
Comment (stephenwf) [inline recipe/0036-composition-from-multiple-images/manifest.json:8]:
```suggestion
            "id": "{{ id.path }}/canvas/p1",
```

Comment (stephenwf) [reply recipe/0036-composition-from-multiple-images/manifest.json]: Great, that's working: https://codesandbox.io/s/0036-composition-from-multiple-images-5h5712?file=/index.html 🎉  thanks!

**Comment by kirschbombe** [comment]
@glenrobson - I've updated all three recipes with the comments from TRC:
- add comma to foldouts recipe title (all instances)
- change uppercase "View" to lowercase in #33 
- extend line highlight in 33
- remove "p" from canvas ids in 35

Should be ready to merge :)

**Pull request merged**